native_client/package.json

```diff
@@ -4,8 +4,7 @@
   "description": "A client binary to access DeepSpeech trained models.",
   "license": "MPL-2.0",
   "author": {
-    "name": "Chris Lord",
-    "email": "contact@chrislord.net"
+    "name": "Mozilla",
```

Comment:
Extra comma is making this fail: https://tools.taskcluster.net/groups/RsTGAEYDSM6wX_zLIrV28g/tasks/QMt9mXSTQIalwP6Z2yUseQ/runs/0/logs/public%2Flogs%2Flive_backing.log#L568

Reply:
Thanks! :)